### ISSUE TITLE
[JOSS] Example Usage

### ISSUE THREAD

**Issue by RMeli** [ur-whitelab/hoomd-tf#197]
I think the Jupyter Notebooks showcasing the functionality of the code are great! They are an extremely valuable addition which will facilitate the use and adoption of `hoomd-tf`. To make them even better, I'd suggest to add some text explaining a bit more what the notebook is about and guiding the user through the most complicated cells, if possible.

Running the notebooks with the latest version of the code (be8b299cc8c62add8fff2ff719713b41814cc6ff) I encountered the following problems:

* Coarse Grained Simulations:
  * `RuntimeError: Corrupt GSD file: CG_tutorial/meth.gsd` on cell # 4
  * Spurious `;` at line `charged = hoomd.group.all();` in cell # 4? [Looks like C++ muscle memory ;) ]
* EDS biasing using HTF:
  * `nglview` should be listed as optional dependency at the top of the notebook instead than before cell # 4?
  * `ValueError: Found no dependence on positions or neighborsso forces cannot be computed` 
* Multidimensional EDS biasing:
  * `ValueError: Found no dependence on positions or neighborsso forces cannot be computed` on cell # 3
* Particles in Harmonic Well:
  * `ValueError: Found no dependence on positions or neighborsso forces cannot be computed` on cell # 2

I think the solution for the `ValueError: Found no dependence on positions or neighborsso forces cannot be computed` is the following (see https://github.com/ur-whitelab/hoomd-tf/pull/183#pullrequestreview-436907018 and PR #186 ):

```diff
-    force = graph.compute_forces(energy)
+    force = graph.compute_forces(energy, positions=True)
```

---
[JOSS review](https://github.com/openjournals/joss-reviews/issues/2367)

**Comment by whitead** [comment]
Thanks for pointing out these issues. Some of these notebooks are indeed out of date and not that descriptive. Hopefully I've addressed this in PR #198

**Comment by malramsay64** [comment]
Changes are great. LGTM
